tests/e2e.sh

```diff
@@ -30,21 +30,46 @@ TESTS_TARGETS=($(bazel query 'tests(//tests/e2e/tests/...)'))
 FAILURE_COUNT=0
 SUMMARY='Tests Summary'
 
+cd ${ROOT}
+declare -A pid2testname
+declare -A pid2logfile
 
 for T in ${TESTS_TARGETS[@]}; do
```

Comment:
Could you make this configurable ie run the test sequentially by default and run in parallel when a flag is passed.